portal/src/assets/style/_mobile.scss

```diff
@@ -712,4 +718,118 @@ hr {
   .block-section-header-desktop {
     display: none;
   }
+  .block-section__content {
+    padding: 0 1rem 1rem 1rem;
```

Comment:
Hi @mul53 . The bars on other pages are not centred because of this code. 
![image](https://user-images.githubusercontent.com/14008461/70397528-e1cd9b00-1a77-11ea-9685-dc6d70eb67e9.png)
Maybe adding an id to only the block-section__content on your page and use that as the identity for this styling change.

Reply:
Fixed

Reply:
Cool. Thanks @mul53 !